academic/import_bibtex.py

```diff
@@ -103,7 +105,12 @@ def parse_bibtex_entry(
         log.error(f'Invalid date for entry `{entry["ID"]}`.')
 
     page.fm["date"] = "-".join([year, month, day])
-    page.fm["publishDate"] = timestamp
+    if "publishDate" in page.fm:
```

Comment:
Would this not always be true as I recall the new system is using `hugo new` to autofill `publishDate`?

Reply:
Ah yes, I wrote this code before `hugo new` was being called. Will update PR.